plugins/generator-1.17.1/forge-1.17.1/procedures/explode.java.ftl

```diff
@@ -1,2 +1,2 @@
 if(world instanceof Level _level && !_level.isClientSide())
-	_level.explode(null,(int)${input$x},(int)${input$y},(int)${input$z},(float)${input$power}, Explosion.BlockInteraction.${field$mode!"BREAK"});
\ No newline at end of file
+	_level.explode(null,(int)${input$x},(int)${input$y},(int)${input$z},${opt.toFloat(input$power)}, Explosion.BlockInteraction.${field$mode!"BREAK"});
```

Comment:
I noticed that this method accepts `double` values for x y z. Would it be fine to remove the casts to `int`?

Reply:
If it accepts double, I think it is fine, this would also allow more targeted explosions :D